packages/ui/src/components/select/NetworkSelection.tsx

```diff
@@ -14,31 +21,179 @@ const NetworkSelection = () => {
+  const renderNetworks = useCallback(
+    (allowedNetworks: string[]) =>
+      networksToShow
+        .filter(([networkName]) => allowedNetworks.includes(networkName))
+        .map(([networkName, { logo }]) => (
+          <MenuItemStyled
+            key={networkName}
+            value={networkName}
+          >
+            <ImgStyled
+              alt={`network-logo-${networkName}`}
+              src={logo}
+            />
+            <ItemNameStyled>{networkName}</ItemNameStyled>
+          </MenuItemStyled>
+        )),
+    [networksToShow]
+  )
+
   if (!selectedNetwork) {
     return null
   }
 
   return (
-    <Select
-      sx={{
-        maxWidth: '9.875rem',
-        marginLeft: '1rem'
-      }}
-      inputSize="large"
+    <SelectStyled
+      IconComponent={HiOutlineChevronDown}
       value={selectedNetwork}
-      onChange={handleNetworkSelection}
-      menuItems={networksToShow.map(([name, { logo }]) => ({
-        value: name,
-        logo
-      }))}
-    />
+      autoWidth={true}
+      onChange={(event) => handleNetworkSelection(event as SelectChangeEvent<string>)}
+      MenuProps={MenuPropsStyles}
+    >
+      <ListSubheader>Polkadot & Parachains</ListSubheader>
+      {renderNetworks(polkadotNetworksAndParachains)}
+      <ListSubheader>Kusama & Parachains</ListSubheader>
+      {renderNetworks(kusamaNetworksAndParachains)}
+      <ListSubheader>Testnets</ListSubheader>
+      {renderNetworks(testChains)}
+    </SelectStyled>
   )
 }
 
+const MenuPropsStyles = {
+  sx: {
+    marginTop: '.75rem',
+
+    '.MuiPaper-root': {
+      marginTop: '2px',
+      boxShadow: 'none',
+      border: 'none',
+      outline: `1px solid ${theme.custom.text.borderColor}`,
+
+      ul: {
+        border: 'none'
+      }
+    },
+
+    '.MuiMenuItem-root': {
+      maxWidth: '100%',
+      padding: '0.75rem'
+    },
+
+    '.MuiList-root': {
+      columnFill: 'balance',
+      padding: 0,
+      border: `1px solid ${theme.custom.text.borderColor}`,
+      borderRadius: '0.5rem'
+    },
+
+    '.MuiListSubheader-root': {
+      columnSpan: 'all'
+    },
+
+    [theme.breakpoints.up('xs')]: {
+      '.MuiList-root': {
+        columns: '160px 2'
+      }
+    },
+
+    [theme.breakpoints.up('sm')]: {
+      '.MuiList-root': {
+        columns: '180px 2'
+      }
+    },
+
+    [theme.breakpoints.up('md')]: {
+      '.MuiList-root': {
+        columns: '200px 2'
+      }
+    }
+  }
+}
+
+const SelectStyled = styled(SelectMui)`
+  display: inline-flex;
+  width: 100%;
+  background: ${({ theme }) => theme.palette.primary.white};
+  outline: 1.5px solid ${({ theme }) => theme.custom.text.borderColor};
+  text-transform: capitalize;
+  border-radius: ${({ theme }) => theme.custom.borderRadius};
+  cursor: pointer;
+
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.md}px) {
+    margin-left: 1rem;
+    max-width: fit-content;
+  }
+
+  .MuiSelect-select {
+    display: flex;
+    padding-right: 2.5rem !important;
+
```

Comment:
reducing the padding here to fit for a bigger logo
```suggestion
padding-top: 0.75rem;
padding-bottom: 0.75rem;

```